examples/simulation/Tutorial1/src/FairTutorialDet1.cxx

```diff
@@ -24,40 +24,16 @@ FairTutorialDet1Geo* FairTutorialDet1::fgGeo = nullptr;
 
 FairTutorialDet1::FairTutorialDet1()
     : FairDetector("TutorialDet", kTRUE, kTutDet)
-    , fTrackID(-1)
-    , fVolumeID(-1)
-    , fPos()
-    , fMom()
-    , fTime(-1.)
-    , fLength(-1.)
-    , fELoss(-1)
-    , fIsInitialised(kFALSE)
     , fFairTutorialDet1PointCollection(new TClonesArray("FairTutorialDet1Point"))
 {}
 
 FairTutorialDet1::FairTutorialDet1(const char* name, Bool_t active)
     : FairDetector(name, active, kTutDet)
-    , fTrackID(-1)
-    , fVolumeID(-1)
-    , fPos()
-    , fMom()
-    , fTime(-1.)
-    , fLength(-1.)
-    , fELoss(-1)
-    , fIsInitialised(kFALSE)
     , fFairTutorialDet1PointCollection(new TClonesArray("FairTutorialDet1Point"))
 {}
 
 FairTutorialDet1::FairTutorialDet1(const FairTutorialDet1& rhs)
```

Comment:
Probably can be default.

Reply:
Using `= default` would make me fear, that it could copy the pointer to the `TClonesArray` just over. Let's please stay explicit here.